database/migrations/2019_09_28_171753_config_fix.php

```diff
@@ -0,0 +1,577 @@
+				'type_range' => STRING,
+				'confidentiality' => '3',
+			],
+			[
+				'key' => 'skip_duplicates',
+				'value' => '0',
+				'cat' => 'Image Processing',
+				'type_range' => BOOL,
+				'confidentiality' => '2',
+			],
+			[
+				'key' => 'lang',
+				'value' => 'en',
+				'cat' => 'Gallery',
+				'type_range' => DISABLED,
+				'confidentiality' => '0',
+			],
+			[
+				'key' => 'layout',
+				'value' => '1',
+				'cat' => 'Gallery',
+				'type_range' => TERNARY,
+				'confidentiality' => '0',
+			],
+			[
+				'key' => 'image_overlay',
+				'value' => '1',
+				'cat' => 'Gallery',
+				'type_range' => BOOL,
+				'confidentiality' => '0',
+			],
+			[
+				'key' => 'default_license',
+				'value' => 'none',
+				'cat' => 'Gallery',
+				'type_range' => STRING_REQ,
+				'confidentiality' => '2',
+			],
+			[
+				'key' => 'small_max_width',
+				'value' => '0',
+				'cat' => 'Image Processing',
+				'type_range' => INT,
+				'confidentiality' => '2',
+			],
+			[
+				'key' => 'small_max_height',
+				'value' => '360',
+				'cat' => 'Image Processing',
+				'type_range' => INT,
+				'confidentiality' => '2',
+			],
+			[
+				'key' => 'medium_max_width',
+				'value' => '1920',
+				'cat' => 'Image Processing',
+				'type_range' => INT,
+				'confidentiality' => '2',
+			],
+			[
+				'key' => 'medium_max_height',
+				'value' => '1080',
+				'cat' => 'Image Processing',
+				'type_range' => INT,
+				'confidentiality' => '2',
+			],
+			[
+				'key' => 'full_photo',
+				'value' => '1',
+				'cat' => 'Gallery',
+				'type_range' => BOOL,
+				'confidentiality' => '0',
+			],
+			[
+				'key' => 'delete_imported',
+				'value' => '1',
+				'cat' => 'Image Processing',
+				'type_range' => BOOL,
+				'confidentiality' => '2',
+			],
+			[
+				'key' => 'Mod_Frame',
+				'value' => '1',
+				'cat' => 'Mod Frame',
+				'type_range' => BOOL,
+				'confidentiality' => '0',
+			],
+			[
+				'key' => 'Mod_Frame_refresh',
+				'value' => '30000',
+				'cat' => 'Mod Frame',
+				'type_range' => INT,
+				'confidentiality' => '0',
+			],
+			[
+				'key' => 'image_overlay_type',
+				'value' => 'desc',
+				'cat' => 'Gallery',
+				'type_range' => 'exif|desc|takedate',
+				'confidentiality' => '0',
+			],
+			[
+				'key' => 'compression_quality',
+				'value' => '90',
+				'cat' => 'Image Processing',
+				'type_range' => INT,
+				'confidentiality' => '2',
+			],
+			[
+				'key' => 'landing_page_enable',
+				'value' => '1',
+				'cat' => 'Mod Welcome',
+				'type_range' => BOOL,
+				'confidentiality' => '0',
+			],
+			[
+				'key' => 'landing_owner',
+				'value' => 'John Smith',
+				'cat' => 'Mod Welcome',
+				'type_range' => STRING,
+				'confidentiality' => '2',
+			],
+			[
+				'key' => 'landing_title',
+				'value' => 'John Smith',
+				'cat' => 'Mod Welcome',
+				'type_range' => STRING,
+				'confidentiality' => '2',
+			],
+			[
+				'key' => 'landing_subtitle',
+				'value' => 'Cat, Dogs & Humans Photography',
```

Comment:
Shouldn't that be _Cat**s**, Dogs & Humans Photography_? :smile:

Reply:
@ildyria only has one cat :wink: